src/validation.cpp

```diff
@@ -3511,9 +3538,9 @@ CBlockIndex * InsertBlockIndex(uint256 hash)
     return pindexNew;
 }
 
-bool static LoadBlockIndexDB(const CChainParams& chainparams)
+bool CChainState::Load(CBlockTreeDB& blocktree)
```

Comment:
The CChainParams are still needed (or at least Consensus::Paras) see https://github.com/bitcoin/bitcoin/pull/9176